docs/docsite/rst/user_guide/playbooks_reuse_roles.rst

```diff
@@ -217,6 +217,8 @@ You can conditionally include a role:
             name: some_role
           when: "ansible_facts['os_family'] == 'RedHat'"
 
+Variables of the role are only available to the role (except if option `public: true` is set, see :ref:`ansible.builtin.include_role_module`).
```

Comment:
ansible bot tells this reference is wrong. Will have to find out how to do it properly.

Reply:
Any hint (from a human) how to do this reference properly?